airbyte-integrations/connectors/source-mssql/src/main/java/io/airbyte/integrations/source/mssql/MssqlSource.java

```diff
@@ -267,7 +277,7 @@ public Set<String> getExcludedInternalNameSpaces() {
   public AirbyteCatalog discover(final JsonNode config) throws Exception {
     final AirbyteCatalog catalog = super.discover(config);
 
-    if (MssqlCdcHelper.isCdc(config)) {
+    if (isCdc(config)) {
```

Comment:
please don't do a static import of functions. It makes it a lot harder to know where the function lives without looking at the imports

Reply:
Ack. I'll make the change.